include/mipi_syst/platform.h

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
this is not a zephyr public API, you should move it under subsystem/logging or subsystem/debug.

Reply:
ok, Thanks